eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -68,27 +70,29 @@ Get-Content -Raw path/to/file.yml | CompatibleConvertFrom-Yaml
 #>
 function CompatibleConvertFrom-Yaml {
   param(
-    [Parameter(Mandatory=$true, ValueFromPipeline=$true)]
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
     [string]$Content
   )
 
   if (!($Content)) {
     throw "Content to parse is a required input."
   }
 
-  # Initialize any variables or checks that need to be done once
-  $yqPresent = Get-Command 'yq' -ErrorAction SilentlyContinue
-  if (-not $yqPresent) {
-    . (Join-Path $PSScriptRoot PSModule-Helpers.ps1)
-    Install-ModuleIfNotInstalled -WhatIf:$false "powershell-yaml" "0.4.1" | Import-Module
+  if ($null -eq $script:yqPresent) {
+    $script:yqPresent = [bool](Get-Command 'yq' -ErrorAction SilentlyContinue)
```

Comment:
Can we still skip yq altogether given that powershell-yaml works?

Reply:
Or would we rather avoid the download if possible. Just seeing as it doesn't exist on windows I feel it may be better to consistently use the same yaml processor for all platforms.

Reply:
I think that Daniel's suggestion is the way to go if we do want to just use the yaml processor from powershell.

Reply:
Yeah if we can achieve more consistency without the reliability tradeoff I think it makes sense. I share the leeriness of the forced installation at a low level but it may be the best trade-off.